be/src/common/config.h

```diff
@@ -852,6 +852,10 @@ CONF_Int64(object_storage_request_timeout_ms, "-1");
 CONF_Strings(fallback_to_hadoop_fs_list, "");
 CONF_Strings(s3_compatible_fs_list, "s3n://, s3a://, s3://, oss://, cos://, cosn://, obs://, ks3://, tos://");
 CONF_mBool(s3_use_list_objects_v1, "false");
+CONF_String(jindo_fs_test_endpoint, "");
+// for jindo test only
+CONF_Strings(jindo_fs_list, "");
+CONF_String(jindo_user, "starrocks");
```

Comment:
Can we put `jindo_fs_test_endpoint` and `jindo_user` in cpp, `config.h`'s parameter is public to user.

Reply:
`jindo_fs_test_endpoint ` is used in UT, it is expected to be read when unit test enable testing for Jindo, like the s3 case.
Remove it from config is ok, but supposed to be somewhere else to inject access info for the UT when enabled of running that case.

`jindo_user ` is use for init jindo store, a fundamental object used by JindoAPI.
As the Jindo C API has no public document, and the behavior is unclear at least for me, So I think it is better to be a tunable one for end used in case of Jindo Service behavior changed.